crates/primitives/state/Cargo.toml

```diff
@@ -12,8 +12,34 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 blockifier = { workspace = true }
+mp-felt = { workspace = true, features = [
+  "parity-scale-codec",
+  "serde",
+  "scale-info",
+] }
+serde = { workspace = true, optional = true }
+serde_with = { workspace = true, optional = true }
```

Comment:
I like to visually separate the optional deps from the other ones with a
```rust
# Optiona
```
comment

It helps readability.


True for all you Cargo.toml

Reply:
Corrected for all the referenced cargos (removing Substrate dependencies comment to avoid having multiple comment lines)